defi/serverless.yml

```diff
@@ -95,6 +95,7 @@ provider:
     R2_SECRET_ACCESS_KEY: ${file(./env.js):R2_SECRET_ACCESS_KEY}
     R2_ENDPOINT: ${file(./env.js):R2_ENDPOINT}
     INDEXA_DB: ${file(./env.js):INDEXA_DB}
+    COINS_DB: ${file(./env.js):COINS_DB}
```

Comment:
this will add the env variable to all functions

either define it only here or in the functions where you want it (much better), not both

Reply:
fixed it by removing it